app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -489,21 +489,6 @@ class BrowserFragment : Fragment(), BackHandler {
                         trackToolbarItemInteraction(it)
                         handleToolbarItemInteraction(it)
                     }
-                    is SearchAction.ToolbarLongClicked -> {
```

Comment:
Can we also delete this action where it is defined?

Reply:
Done!